test/org/sagebionetworks/bridge/validators/ScheduleValidatorTest.java

```diff
@@ -102,4 +102,110 @@ public void activityCorrectlyParsesPublishedSurveyPath() {
         assertEquals("2015-01-27T17:46:31.237Z", ref.getCreatedOn());
     }
     
+    @Test
+    public void recurringSchedulesMustHaveCronTriggerOrInterval() {
+        Schedule schedule = new Schedule();
+        schedule.setScheduleType(ScheduleType.RECURRING);
+        try {
+            Validate.entityThrowingException(validator, schedule);
+        } catch(InvalidEntityException e) {
+            assertEquals("Schedule recurring schedules should have either a cron expression, or an interval", e.getErrors().get("Schedule").get(0));
+        }
+        
+        schedule.setInterval("P1D");
+        try {
+            Validate.entityThrowingException(validator, schedule);
+        } catch(InvalidEntityException e) {
+            assertNull(e.getErrors().get("Schedule"));
+        }
+    }
+    
+    @Test
+    public void rejectsInvalidCronExpression() {
+        Schedule schedule = new Schedule();
+        schedule.setScheduleType(ScheduleType.RECURRING);
+        schedule.setCronTrigger("2 3 a");
+
+        try {
+            Validate.entityThrowingException(validator, schedule);
+        } catch(InvalidEntityException e) {
+            assertEquals("cronTrigger is an invalid cron expression", e.getErrors().get("cronTrigger").get(0));
+        }
+    }
+    
+    /**
+     * It's ambiguous what this means when longer intervals require a time of day as well...the interval, delay, and  
+     * times can all conflict with one another.
+     */
+    @Test
+    public void rejectsInvervalOfLessThanADay() {
+        Schedule schedule = new Schedule();
+        schedule.setScheduleType(ScheduleType.RECURRING);
+        schedule.setInterval("PT30M");
+        
+        try {
+            Validate.entityThrowingException(validator, schedule);
+        } catch(InvalidEntityException e) {
+            assertEquals("interval must be at least one day", e.getErrors().get("interval").get(0));
+        }
+    }
+    
+    @Test
+    public void rejectsDelayOfLessThanADay() {
+        
+    }
```

Comment:
Empty unit test, which is also redundant with delaysWithTimesOfDayMustBeMoreThanOneDayLong()